common/source/docs/common-network.rst

```diff
@@ -0,0 +1,118 @@
+.. _common-network:
+
+===========================
+ArduPilot Network Interface
+===========================
+
+As of firmware 4.5 and later, ArduPilot provides a network interface framework to allow local and wide-area network connections. Several later generation autopilots, like Pixhawk6X and CubePilot CubeRed, provide Ethernet physical layer interfaces, which allows ArduPilot to connect to vehicle peripherals, data servers, and even the wide area network via IP using transport layer UDP or TCP protocols.
+
+Currently, hooks into the normal serial manager within ArduPilot now allow serial protocols not only to be connected to the autopilot via UART connections, but also via network connections. These are currently limited to using Ethernet physical layer.
```

Comment:
Given PPP this "These are currently limited to using Ethernet physical layer." may no longer be true right?

Reply:
The wiki 4.5 for PP has not be done yet....I will be making changes appropriately then

Reply:
Just wanted to capture it as I'm working through it, I'll forget if I try to remember it later.

Reply:
updated for PPP